pkg/wrapper/nmt_wrapper_test.go

```diff
@@ -132,7 +142,8 @@ func generateErasuredData(t *testing.T, numLeaves int, codec rsmt2d.Codec) [][]b
 	return append(raw, erasuredData...)
 }
 
-// this code is copy pasted from the plugin, and should likely be exported in the plugin instead
+// generateRandNamespacedRawData returns random data of length total. Each chunk
+// of random data is of size nidSize + leafSize.
 func generateRandNamespacedRawData(total int, nidSize int, leafSize int) [][]byte {
```

Comment:
FWIW this could be simplified to:

```go
func generateRandNamespacedRawData(numShares int, shareSize int) (data [][]byte) {
	for i := 0; i < numShares; i++ {
		share := make([]byte, shareSize)
		_, err := rand.Read(share)
		if err != nil {
			panic(err)
		}
		data = append(data, share)
	}
	sortByteArrays(data)
	return data
}
```

Reply:
There is just a slight difference and that is the final `data` is sorted both based on namespace ID and their data whereas in the original implementation it is not the case.